app/src/main/java/io/neurolab/main/output/visual/OSCFeedback.java

```diff
@@ -0,0 +1,35 @@
+package io.neurolab.main.output.visual;
+
+import java.net.SocketException;
+
+import io.neurolab.main.output.feedback.Feedback;
+import io.neurolab.settings.FeedbackSettings;
+import com.illposed.osc.OSCPortIn;
+
+public class OSCFeedback extends Feedback {
+
+    private static OSCPortIn receiver;
```

Comment:
Any specific reason to make this static?

Reply:
I did this because it's more of a class variable as OSCPortIn is the receiver which listens for OSC messages; I thought it more of a class property than OSCFeedback's object property (like two OSCFeedback objects can share the same listener and don't need individual listeners). I believe this is why it was done in the desktop repo as well. However, it's not that compelling and we can surely have a workaround with making it an instance variable if you say so?

Reply:
No for now we'll keep it as it is and address it as we progress

Reply:
Btw, Codacy warns this as unused. Is it?

Reply:
@CloudyPadmal yes actually. The whole thing would make sense when we link it up with UI in Android (UI would behave according to the messages/data received). Actually, it is just a base functionality.

Reply:
Ok. got it.